person.rb

```diff
@@ -2,14 +2,19 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
   end
```

Comment:
- [ ] Your add_rental method should look something similar to the below one :arrow_down:

```ruby
  def add_rental(person, date)
    Rental.new(date, book, self)
  end
```